controller/workitem_blackbox_test.go

```diff
@@ -969,6 +969,139 @@ func (s *WorkItem2Suite) TestWI2UpdateSetReadOnlyFields() {
 	})
 }
 
+func (s *WorkItem2Suite) TestWI2UpdateWorkItemType() {
```

Comment:
[SUGGESTION] There should be a test case to capture multiple times work item type change on same workitem.

Reply:
@DhritiShikhar the test here https://github.com/fabric8-services/fabric8-wit/pull/2202/files#diff-1999ee68a49d5d171f3f4b3e1c4ccd04R155 is not the same but it shows that we essentially can convert back and forth. It doesn't use the same work item for this though.